charts/seed-monitoring/charts/grafana/dashboards/owners/node-pool-dashboard.json

```
@@ -884,7 +884,7 @@
           "refId": "G"
         },
         {
-          "expr": "  count by (node) (kube_pod_info{type=\"shoot\"})\n+ on (node) group_right ()\n  sum by (label_worker_gardener_cloud_pool, node) (\n    0 * kube_node_labels{label_worker_gardener_cloud_pool=~\"$worker_group\"}\n  )",
+          "expr": "  sum without (kubernetes_io_hostname) (label_replace(sum (kubelet_running_pods) by (kubernetes_io_hostname), \"node\",\"$1\",\"kubernetes_io_hostname\", \"(.+)\")\n  )",
```

I recall monitoring colleagues were asking the queries in the dashboards to be formatted by simply pasting the query in prometheus and formatting it there.

```suggestion
          "expr": "sum without (kubernetes_io_hostname) (\n  label_replace(\n    sum by (kubernetes_io_hostname) (kubelet_running_pods),\n    \"node\",\n    \"$1\",\n    \"kubernetes_io_hostname\",\n    \"(.+)\"\n  )\n)",
```

---

How to decide the no. of spaces?

---

I only copied this query from the PR, pasted in the prometheus query, formatted it there by clicking the format button and finally pasted it back in the panel in grafana. I didn't do this manually and didn't decide on any no. of spaces.

---

Thanks 👍